src/main/flight/gps_rescue.h

```diff
@@ -37,6 +37,8 @@ typedef struct gpsRescue_s {
     uint8_t sanityChecks;
     uint8_t allowArmingWithoutFix;
     uint8_t useMag;
+    uint16_t targetLandingAltitudeM; //meters
```

Comment:
Now you've removed all means to set these - they should at least be exposed in CLI through `settings.c`.

Reply:
ok

Reply:
I meant both of them. :wink: